elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -282,6 +288,57 @@
 
     static final ObjectTypeAttributeDefinition CERTIFICATE_REVOCATION_LIST = new ObjectTypeAttributeDefinition.Builder(ElytronDescriptionConstants.CERTIFICATE_REVOCATION_LIST, PATH, RELATIVE_TO, MAXIMUM_CERT_PATH)
             .setRequired(false)
+            .setAlternatives(ElytronDescriptionConstants.ONLINE_CERTIFICATE_STATUS)
+            .setRestartAllServices()
+            .build();
+
+    private static final SimpleAttributeDefinition RESPONDER = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.RESPONDER, ModelType.STRING, true)
+            .setAllowExpression(true)
+            .setRestartAllServices()
+            .build();
+
+    private static final SimpleAttributeDefinition OCSP_TRUSTSTORE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.KEY_STORE, ModelType.STRING, true)
+            .setAllowExpression(true)
+            .setCapabilityReference(KEY_STORE_CAPABILITY, TRUST_MANAGER_CAPABILITY)
+            .setRestartAllServices()
+            .build();
+
+    private static final SimpleAttributeDefinition ACCEPT_UNKNOWN = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.ACCEPT_UNKNOWN, ModelType.BOOLEAN, true)
+            .setAllowExpression(true)
+            .setDefaultValue(new ModelNode(false))
+            .setRestartAllServices()
+            .build();
+
+    private static final SimpleAttributeDefinition ACCEPT_WHEN_STATUS_UNAVAILABLE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.ACCEPT_WHEN_STATUS_UNAVAILABLE, ModelType.BOOLEAN, true)
+            .setAllowExpression(true)
+            .setDefaultValue(new ModelNode(false))
+            .setRestartAllServices()
+            .build();
+
+    private static final SimpleAttributeDefinition CONNECTION_TIMEOUT = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.CONNECTION_TIMEOUT, ModelType.INT, true)
+            .setAllowExpression(true)
+            .setRestartAllServices()
+            .build();
```

Comment:
Maybe we should set default value so customer can see in CLI/Web Console what the actual default value is?

Reply:
fixed